project-templates/ecs/basic/terraform-aws-example-microservice-infra/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+   cloud {
+    organization = "{{YOUR_TERRAFORM_ORGANIZATION_NAME}}"
```

Comment:
I think instructions should be added to the README to update this value with the user's actual TFC org name.

Reply:
Nice catch! I fixed it!
<img width="1160" alt="image" src="https://github.com/hashicorp/waypoint-examples/assets/1695478/8ed90519-1ff0-4dec-90b7-0b553bb0c993">